client/src/app/[site]/components/shared/Map/hooks/useMapInstance.ts

```
@@ -0,0 +1,141 @@
+import { useEffect, useRef, MutableRefObject } from "react";
+import Map from "ol/Map";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)